### DISCUSSION TITLE
Welcome to LGWebOSRemote Discussions!

### DISCUSSION THREAD

**Discussion by klattimer** [klattimer/LGWebOSRemote#138] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by law94air1976** [comment]
Ive run into a problem - when I now press the power off button on my remote the TV screen goes off but it does not power off - there is no click sound that you can hear and when I press the power button again the screen comes on immediately.  Ive uninstalled the LGTV python script and restarted the mac and unplugged the TV but I still have the issue - How do I get back the power off feature of my TV?

**Comment by caniswim** [comment]
Is it possible to change the backlight value?

**Comment by clownbreath** [comment]
This is so awesome! Thank you.

Everything is working for me except lgtv --name MyTV --ssl setTVChannel XXX

This is the output of getTVChannel what should I put on the command line? Thank you.

lgtv --name MyTV --ssl getTVChannel
{"type": "response", "id": "channels_0", "payload": {"returnValue": true, "channelTypeName": "Terrestrial Digital TV", "channelNumber": "5-4", "isChannelChanged": false, "channelName": "Quest", "isInteractiveRestrictionChannel": false, "isSkipped": false, "isReplaceChannel": false, "isLocked": false, "isHEVCChannel": false, "channelModeId": 0, "isInvisible": false, "isScrambled": false, "signalChannelId": "1_35_5_4_0_0_0", "physicalNumber": 35, "hybridtvType": null, "isDescrambled": false, "channelModeName": "Terrestrial", "channelId": "1_35_5_4_0_0_0", "favoriteGroup": null, "channelTypeId": 1, "isFineTuned": false, "dualChannel": {"dualChannelTypeName": null, "dualChannelNumber": null, "dualChannelId": null, "dualChannelTypeId": 255}}}
{"closing": {"code": 1000, "reason": ""}}

**Reply by clownbreath** [reply]
lgtv --name MyTV --ssl getSystemInfo
{"type": "response", "id": "0", "payload": {"returnValue": true, "features": {"dvr": false}, "receiverType": "ATSC", "modelName": "50UQ8000AUB", "serialNumber": "206MXWE49993", "programMode": false}}
{"closing": {"code": 1000, "reason": ""}}

**Reply by clownbreath** [reply]
It worked great with youtube and browser! :)
